### PR TITLE
Fix ccw test cases failures in s390x

### PR DESCRIPTION
**Pull request by chunfuwen**
attach_disk,attach_controller,attach_rng is supported on s390x, so it is positive cases

### CODE REVIEW

**Comment by chunfuwen** [comment]
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.hotplug.attach_disk.positive_test.start_vm
PASS 2-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.hotplug.attach_controller.positive_test.start_vm
PASS 3-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.hotplug.attach_rng.positive_test.start_vm
PASS 4-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.coldplug.attach_no_exist_disk.negative_test.start_vm
PASS 5-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.coldplug.attach_disk.positive_test.start_vm
PASS 6-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.coldplug.attach_controller.positive_test.start_vm
PASS 7-type_specific.io-github-autotest-libvirt.virtual_disks.ccw_addr.coldplug.attach_rng.positive_test.start_vm

**Comment by chunfuwen** [comment]
Failed failures:

TestFail: unexpected exception happen: Expect should fail with one of ['cannot use CCW address type for device'], but failed with:error: Failed to attach device from /var/tmp/xml_utils_temp_ordp7bs9.xmlerror: internal error: unable to execute QEMU command 'object-add': Could not open '/dev/hwrng': Permission denied&#10;
TestFail: unexpected exception happen: VM started unexpectedly.&#10;

**Comment by chunfuwen** [comment]
@smitterl  please help take a look

File: libvirt/tests/cfg/virtual_disks/virtual_disks_ccw_addr.cfg [hunk 1]
```diff
@@ -37,7 +38,7 @@
             addr_attrs = "{'type': 'ccw', 'cssid': '0x0', 'ssid': '0x0', 'devno': '0x0000'}"
         - attach_rng:
             backend_device = "rng"
-            backend_dev = "/dev/hwrng"
+            backend_dev = "/dev/random"
```
Comment (smitterl) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_ccw_addr.cfg:41]: There should be a /dev/hwrng device also on s390x but this test suite isn't about testing the backend, so I think it's okay to use /dev/random instead.

**Review by smitterl — APPROVED** [review]
Thank you!

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**